Source/WebCore/page/ChromeClient.h

```diff
@@ -668,13 +668,13 @@ class ChromeClient {
 
     virtual void textReplacementSessionUpdateStateForReplacementWithID(const UnifiedTextReplacement::SessionID&, UnifiedTextReplacement::ReplacementState, const UnifiedTextReplacement::ReplacementID&) { }
 
-    virtual void removeTextIndicatorStyleForID(const UnifiedTextReplacement::SessionID&) { }
+    virtual void removeTextAnimationForID(const UnifiedTextReplacement::SessionID&) { }
```

Comment:
`removeWritingToolsTextAnimationForSessionWithID`?

Reply:
So we went with not using writing Tools for this because I also use them for Safari.

Reply:
ok, then in that case I think it should be `removeTextAnimationForSessionWithID` (for consistency + clarity)